data/json/uncraft/armor/torso.json

```
@@ -1,4 +1,43 @@
 [
+  {
+    "result": "tacjacket",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "skill_used": "tailor",
+    "difficulty": 2,
+    "time": "4 m 30 s",
+    "qualities": [ { "id": "FABRIC_CUT", "level": 1 }, { "id": "CUT", "level": 2 } ],
+    "components": [
+      [ [ "zipper_short_plastic", 3 ] ],
+      [ [ "zipper_long_plastic", 1 ] ],
+      [ [ "snapfastener", 5 ] ],
```

```suggestion
      [ [ "snapfastener_steel", 5 ] ],
```

This ID doesn't exist